coordinator/src/services/translation/common/index.ts

```diff
@@ -174,3 +164,33 @@ export function getNumericValueAsString(numericValue: string | number | Lossless
     return numericValue.toString()
   }
 }
+
+function isStringParameter(parameter: fhir.Parameter): parameter is fhir.StringParameter {
+  return (parameter as fhir.StringParameter).valueString !== undefined
+}
+
+function isIdentifierParameter(parameter: fhir.Parameter): parameter is fhir.IdentifierParameter {
+  return (parameter as fhir.IdentifierParameter).valueIdentifier !== undefined
+}
+
+export function getStringParameterByName(
```

Comment:
Do you think these kind of low level functions would benefit from some unit tests? I know are currently relying on the integration message tests to valdate the functionality, but these definitely look like something that could be tested in isolation

Reply:
I'll add some unit tests around these